proto/operator.proto

```diff
@@ -106,29 +106,50 @@ message Notification {
   }
 }
 
-enum NotificationType {
-  NOTIFICATION_TYPE_UNSPECIFIED = 0;
-  NOTIFICATION_TYPE_CHANNEL_OPENING = 1;
-  NOTIFICATION_TYPE_CHANNEL_OPENED = 2;
-  NOTIFICATION_TYPE_CHANNEL_CLOSED = 3;
-  NOTIFICATION_TYPE_CHALLENGE_STARTED = 4;
-  NOTIFICATION_TYPE_CHALLENGE_CLEARED = 5;
-  NOTIFICATION_TYPE_CHALLENGE_FINISHED = 6;
-  NOTIFICATION_TYPE_MARGIN_UPDATED = 7;
-}
-
-enum ChannelStatus {
-  CHANNEL_STATUS_UNSPECIFIED = 0;
-  CHANNEL_STATUS_OPENING = 1;
-  CHANNEL_STATUS_OPEN = 2;
-  CHANNEL_STATUS_CHALLENGING = 3;
-  CHANNEL_STATUS_CLOSED = 4;
+enum ClearingEvent {
```

Comment:
`CLEARING_EVENT_AGREED_ON_POSTFUND` and `CLEARING_EVENT_PROCESS_MARGIN_CALL` are missing.